src/Microsoft.Framework.Logging/Utils/TypeNameHelper.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using System.Text;
+
+namespace Microsoft.Framework.Logging
```

Comment:
Thanks. I'll move it to `Microsoft.Framework.Logging.Internal`. Hope that is good